exercism/hamming/hamming.spec.js

```diff
@@ -1,40 +1,40 @@
 var Hamming = require('./hamming');
 
-describe('Hamming', function () {
+describe('Hamming', function() {
   var hamming = new Hamming();
 
-  it('no difference between identical strands', function () {
+  it('no difference between identical strands', function() {
```

Comment:
I would write "Returns 0 if strands are identical" here, add more specificity to what it is actually testing